src/Solvers/Random.jl

```diff
@@ -1,8 +1,54 @@
+using OptimizedEinsum: removedsize, removedsize_noise, ssa_path_cost, ContractionPath
```

Comment:
```suggestion
using OptimizedEinsum: removedsize, ssa_path_cost, ContractionPath
```

Reply:
I added this together with some minor fixes due to the last suggestions in a new commit.